tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Services/CreateOrderServiceTests.cs

```diff
@@ -1,159 +1,159 @@
-        }
-
-        [Test]
-        public async Task CreateAsync_OrderValidationFailsWithEmptyOrganisationId_ReturnFailureResult()
-        {
-            var context = CreateOrderServiceTestContext.Setup();
-
-            var sut = context.CreateOrderService;
-
-            var request = CreateOrderRequestBuilder
-                .Create()
-                .WithOrganisationId(Guid.Empty)
-                .Build();
-            
-
-            var actual = await sut.CreateAsync(request);
-
-            var expected = Result.Failure<string>(new List<ErrorDetails> {new ErrorDetails("OrganisationIdRequired", "OrganisationId") });
-            
-            actual.Should().Be(expected);
-        }
-
-        [Test]
-        public async Task CreateAsync_OrderValidationFailsWithEmptyDescription_ReturnFailureResult()
-        {
-            var context = CreateOrderServiceTestContext.Setup();
-
-            var sut = context.CreateOrderService;
-
-            var request = CreateOrderRequestBuilder
-                .Create()
-                .WithDescription(null)
-                .Build();
-
-            var actual = await sut.CreateAsync(request);
-
-            var expected = Result.Failure<string>(new List<ErrorDetails> { new ErrorDetails("OrderDescriptionRequired", "Description") });
-
-            actual.Should().Be(expected);
-        }
-
-        [Test]
-        public async Task CreateAsync_OrderValidationFailsWithLongDescription_ReturnFailureResult()
-        {
-            var context = CreateOrderServiceTestContext.Setup();
-
-            var sut = context.CreateOrderService;
-
-            var request = CreateOrderRequestBuilder
-                .Create()
-                .WithDescription("This description is too long to be processed, 1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890")
-                .Build();
-
-            var actual = await sut.CreateAsync(request);
-
-            var expected = Result.Failure<string>(new List<ErrorDetails> { new ErrorDetails("OrderDescriptionTooLong", "Description") });
-
-            actual.Should().Be(expected);
-        }
-    }
-
-    internal sealed class CreateOrderServiceTestContext
-    {
-        private CreateOrderServiceTestContext()
-        {
-            OrderRepositoryMock = new Mock<IOrderRepository>();
-            OrderRepositoryMock.Setup(x => x.CreateOrderAsync(It.IsAny<Order>())).ReturnsAsync("OrderId");
-            CreateOrderService = new CreateOrderService(OrderRepositoryMock.Object);
-        }
-
-        internal Result OrderValidationResult { get; set; } = Result.Success();
-
-        internal Mock<IOrderRepository> OrderRepositoryMock { get; set; }
-
-        internal CreateOrderService CreateOrderService { get; }
-
-        public static CreateOrderServiceTestContext Setup()
-        {
-            return new CreateOrderServiceTestContext();
-        }
-    }
-}
+﻿using System;

```

Comment:
Was this just a line ending change?

Reply:
Yes, what is the preference. CRLF or LF (it interchanges throughout)